implementation/src/main/java/io/smallrye/config/ConfigValueConfigSourceInterceptor.java

```diff
@@ -0,0 +1,25 @@
+package io.smallrye.config;
+
+import javax.annotation.Priority;
+
+import org.jboss.logging.Logger;
+
+@Priority(100)
+public class ConfigValueConfigSourceInterceptor implements ConfigSourceInterceptor {
+    private static final Logger LOG = Logger.getLogger("io.smallrye.config");
+
+    @Override
+    public ConfigValue getValue(final ConfigSourceInterceptorContext context, final String name) {
+        final ConfigValue configValue = context.proceed(name);
+
+        final String key = configValue.getName();
+        final String value = configValue.getValue();
+        final String configSource = configValue.getConfigSourceName();
+        final int lineNumber = configValue.getLineNumber();
+
+        LOG.infov("The key {0} was loaded from {1} with the value {2} with additional information: {3}", key,
```

Comment:
Maybe this text needs updating?  Also FWIW, it's recommended to use `infof` rather than `infov`.